components/_helpers.js

```diff
@@ -67,10 +67,15 @@ function normalizeDate(dateObject, hours = 0, minutes = 0, seconds = 0, millisec
   return dateObject;
 }
 
+function getNum(str) {
```

Comment:
imho we don't need this function.
And better name is `parseInt10` if you will decide to keep it